test/color/test_ycbcr.py

```diff
@@ -0,0 +1,248 @@
+import pytest
+
+import kornia
+import kornia.testing as utils  # test utils
+
+import cv2
+import numpy as np
+
+import torch
+from torch.autograd import gradcheck
+from torch.testing import assert_allclose
+
+
+class TestRgbToYcbcr:
+
+    def test_rgb_to_ycbcr(self):
```

Comment:
@justanhduc check da793cd48e17eca7e78b042240a9464065611d9d. Cuda test is enabled now, from root run `make test-gpu` or alternatively `pytest  test/color/test_ycbcr.py --typetest cuda`


```python
from test.common import device

def test_rgb_to_ycbcr(self, device):
    data = torch.tensot(...).to(device)

```